### ISSUE TITLE
Unable to run sarek workflow in AWS health omics 

### ISSUE THREAD

**Issue by DimitrisZisis** [nf-core/sarek#1458]
### Description of the bug

I was following the tutorial https://catalog.us-east-1.prod.workshops.aws/workshops/76d4a4ff-fe6f-436a-a1c2-f7ce44bc5d17/en-US/workshop/test-workflow and I managed to create my workflow in AWS Helathomics. Everything is correctly done but when I am trying to run the workflow in healthomics it is not working  
 I am receiving strange kinds of errors related to the input parameters or samplesheet.I changed the parameters and I removed the -- so now I am getting the following. Do you have any idea what can be wrong? 

### Command used and terminal output

```console
WARNING: Could not load nf-core/config profiles: https://raw.githubusercontent.com/nf-core/configs/master/nfcore_custom.config
WARNING: Could not load nf-core/config/sarek profiles: https://raw.githubusercontent.com/nf-core/configs/master/pipeline/sarek.config
Launching `sarek/main.nf` [prickly_mirzakhani] DSL2 - revision: 48ed053221
Module path must start with / or ./ prefix -- Offending module: plugin/nf-validation,  -- Check script 'sarek/main.nf' at line: 79 or see '.nextflow.log' file for more details
Nextflow engine failed
```

**Comment by asp8200** [comment]
@DimitrisZisis : Thanks for reporting this. I know that Sarek runs fine over awsbatch, so I guess it should also be possible to get it running over AWS Healthomics.

Did you manage to get through the tutorial using `scrnaseq`?

I suggest that you come on over on nf-core's Slack channel for Sarek, and we can try to see if we can help you. Cheers

**Comment by DimitrisZisis** [comment]
Yes i followed all the steps of the tutorial for scrnaseq I did exactly what they described but at this point, I have this error. It seems to have a problem with `plugin/nf-validation` but I don't know why. Do you think that this can be related to IAM role? apart from the zip and what is described in the tutorial is it necessary to upload something else in Healthomics ? 

Regarding Slack channel, i tried to register and get connected but it is impossible. They support only 2 channels (seqera.io and pbic...) and Google doesn't work!

**Comment by asp8200** [comment]
https://nfcore.slack.com/channels/sarek

**Comment by DimitrisZisis** [comment]
Yes but the above link and any other connection link doesn't work. Ican get the 6 digits code in my email

**Comment by DimitrisZisis** [comment]
after carebully testing sarek on my computer, it seems that one of the problems is the version of nextflow that AWS healthomics uses. I installed the same version (version 22.04.0) and I get an error about the same nf-validation plugin. 

```
N E X T F L O W  ~  version 22.04.0
Launching `[https://github.com/nf-core/sarek`](https://github.com/nf-core/sarek%60) [spontaneous_kowalevski] DSL2 - revision: 6aeac929c9 [3.4.0]
Cannot find latest version of nf-validation plugin
```

**Comment by FriederikeHanssen** [comment]
Did you press 'Join' in the top right corner? https://nf-co.re/

**Comment by DimitrisZisis** [comment]
Yes, i did !!! In the section Slack after clicking the connection with google or apple doesn't work and the email section supports only accounts with domains : qbic.uni-tuebingen.de, seqera.io

**Comment by FriederikeHanssen** [comment]
@mribeirodantas can you check what is going on there?

**Comment by DimitrisZisis** [comment]
someone fixed it and i am able to connect ! Thank you! Lets see if we will solve the real problems related to nf-core and AWS healthomics. One basic problem I ve noticed is that the AWS uses a very old version of Nextflow

**Comment by FriederikeHanssen** [comment]
Is this resolved?

**Comment by moe1619** [comment]
i am having the same issue. was this ever resolved?

**Comment by FriederikeHanssen** [comment]
hi! Healthomics is sometimes behind on supported nextflow versions. We have no influence from our end on the version that they support. For the latest on their support I would ask in the slack channel on the nf-core/slack that is linked above
